### PR TITLE
feat: add remix guide to deploy a contract

### PR DESCRIPTION
**Pull request by Megumiiiiii**

### CODE REVIEW

File: src/content/docs/guides/deploy-a-contract.md [hunk 1]
```diff
@@ -43,7 +43,50 @@ Coming soon!
...
+
+- Change the Compiler version to `0.7.0+commit.9e61f92b`
+
+![new file](~/assets/content/docs/guide-contract/compiler.png)
+
+- Then compile it.
+
+### 4. Deploy
+
+- Change the Enviroment to `Injected Provider`
```
Comment (d1onys1us) [inline src/content/docs/guides/deploy-a-contract.md:81]:
```suggestion
- Change the Environment to `Injected Provider`
```

File: src/content/docs/guides/deploy-a-contract.md [hunk 2]
```diff
@@ -43,7 +43,50 @@ Coming soon!
...
+
+![new file](~/assets/content/docs/guide-contract/provider.png)
+
+- Then click `transact
```
Comment (d1onys1us) [inline src/content/docs/guides/deploy-a-contract.md:85]:
```suggestion
- Then click `transact`
```

File: src/content/docs/guides/deploy-a-contract.md [hunk 3]
```diff
@@ -43,7 +43,50 @@ Coming soon!
...
+
+![new file](~/assets/content/docs/guide-contract/transact.png)
+
+- Last, verify the Smart Contract using [Blockscout](/guides/verify-a-contract/#verify-a-contract-with-hardhat-or-other-alternatives)
```
Comment (d1onys1us) [inline src/content/docs/guides/deploy-a-contract.md:89]:
```suggestion
- Finally, verify the smart contract using [Blockscout](/guides/verify-a-contract/#verify-a-contract-with-hardhat-or-other-alternatives)
```

File: src/content/docs/guides/deploy-a-contract.md [hunk 4]
```diff
@@ -43,7 +43,50 @@ Coming soon!
...
+https://remix.ethereum.org
+
+### 2. Create a new `.sol` file
+
+![new file](~/assets/content/docs/guide-contract/newfile.png)
+
+- Give it any name, for example `Counter.sol`.
+- Fill with this example code:
+
+```py
```
Comment (d1onys1us) [inline src/content/docs/guides/deploy-a-contract.md:57]:
```suggestion
```solidity
```

**Review by d1onys1us — COMMENTED** [review]
few comments. also, could you move the images into the folder name `src/assets/content/docs/guides/*`? this is so it follows convention.

otherwise, looks great, thank you for the contribution!

**Comment by Megumiiiiii** [comment]
alright,, wait

**Review by d1onys1us — APPROVED** [review]

**Pull request merged**